### PR TITLE
chore: cron workflow tidying

### PR DESCRIPTION
**Pull request by nicktrn**
## What does this PR do?

Did some cron + curl tidying while reading through the codebase. Pasted crontab.guru output for easy read and verify. The curl flags will reduce unecessary progress meter output in the logs, while retaining previous --fail functionality.

Before:
pre

After:
post

## Requirement/Documentation

N/A

## Type of change

  - [x] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

Cron schedule step values are [supported by Github](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule:~:text=/-,Step%20values,-20/15%20*%20*%20*%20*) and the curl flags are standard.

## Mandatory Tasks

  - [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I have read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code follows the style guidelines of this project
- I have commented my code, particularly in hard-to-understand areas
- I have checked if my PR needs changes to the documentation

### CODE REVIEW

File: .github/workflows/cron-bookingReminder.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ on:
   # "Scheduled workflows run on the latest commit on the default or base branch."
   # — https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#schedule
   schedule:
-    # Runs “At minute 0, 15, 30, and 45.” (see https://crontab.guru)
-    - cron: "0,15,30,45 * * * *"
+    # Runs “At every 15th minute.” (see https://crontab.guru)
+    - cron: "0/15 * * * *"
```
Comment (nicktrn) [inline .github/workflows/cron-bookingReminder.yml:8]: Simpler and easier to edit in the future

File: .github/workflows/cron-bookingReminder.yml [hunk 2]
```diff
@@ -20,4 +20,4 @@ jobs:
             -X POST \
             -H 'content-type: application/json' \
             -H 'authorization: ${{ secrets.CRON_API_KEY }}' \
-            --fail
+            -sSf
```
Comment (nicktrn) [inline .github/workflows/cron-bookingReminder.yml:23]: --silent --show-error --fail

Hide progress while showing and failing on errors

File: .github/workflows/cron-downgradeUsers.yml [hunk 3]
```diff
@@ -5,7 +5,7 @@ on:
   # "Scheduled workflows run on the latest commit on the default or base branch."
   # — https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#schedule
   schedule:
-    # Runs “Every month at 1st (see https://crontab.guru)
+    # Runs “At 00:00 on day-of-month 1.” (see https://crontab.guru)
```
Comment (nicktrn) [inline .github/workflows/cron-downgradeUsers.yml:8]: Be explicit for people unfamiliar with cron

**Review by nicktrn — COMMENTED** [review]
Ready for review

**Comment by nicktrn** [comment]
Just rebased. Very minor but still relevant change.

File: .github/workflows/cron-scheduleEmailReminders.yml [hunk 4]
```diff
@@ -4,8 +4,8 @@ on:
   # "Scheduled workflows run on the latest commit on the default or base branch."
   # — https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#schedule
   schedule:
-    # Runs “At minute 0, 15, 30, and 45.” (see https://crontab.guru)
-    - cron: "0,15,30,45 * * * *"
+    # Runs “At every 15th minute.” (see https://crontab.guru)
+    - cron: "0/15 * * * *"
```
Comment (hariombalhara) [inline .github/workflows/cron-scheduleEmailReminders.yml:8]: Same here

File: .github/workflows/cron-scheduleWhatsappReminders.yml [hunk 5]
```diff
@@ -4,8 +4,8 @@ on:
   # "Scheduled workflows run on the latest commit on the default or base branch."
   # — https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#schedule
   schedule:
-    # Runs “At minute 0, 15, 30, and 45.” (see https://crontab.guru)
-    - cron: "0,15,30,45 * * * *"
+    # Runs “At every 15th minute.” (see https://crontab.guru)
+    - cron: "0/15 * * * *"
```
Comment (hariombalhara) [inline .github/workflows/cron-scheduleWhatsappReminders.yml:8]: Here too

File: .github/workflows/cron-stale-issue.yml [hunk 6]
```diff
@@ -8,7 +8,7 @@ on:
   # "Scheduled workflows run on the latest commit on the default or base branch."
   # — https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#schedule
   schedule:
-    # Runs every day (see https://crontab.guru)
+    # Runs “At 00:00.” (see https://crontab.guru)
```
Comment (hariombalhara) [inline .github/workflows/cron-stale-issue.yml:11]:
```suggestion
    # Runs “At 00:00.” everyday (see https://crontab.guru)
```

File: .github/workflows/submodule-sync.yml [hunk 7]
```diff
@@ -1,6 +1,7 @@
 name: Submodule Sync
 on:
   schedule:
+    # Runs “At minute 15 past every 4th hour.” (see https://crontab.guru)
```
Comment (hariombalhara) [inline .github/workflows/submodule-sync.yml:4]:
```suggestion
    # Runs "At minute 15 past every 4th hour." (see https://crontab.guru)
```

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Thanks @nicktrn 🙏 I have left some comments.

**Comment by nicktrn** [comment]
Thanks for reviewing. I have changed it to `*/15` which is of course equivalent.

On a side note:
Unsure which cron this wouldn't work with. Definitely works with Github cron. The [example for step values](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule:~:text=/-,Step%20values,-20/15%20*%20*%20*%20*) even uses this syntax, i.e. `20/15`, which crontab.guru also complains about. Maybe the guru is wrong here. (It happily accepts `0-0/15`)

**Review by hariombalhara — APPROVED** [review]

**Comment by hariombalhara** [comment]
> Thanks for reviewing. I have changed it to `*/15` which is of course equivalent.
> 
> On a side note: Unsure which cron this wouldn't work with. Definitely works with Github cron. The [example for step values](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule:~:text=/-,Step%20values,-20/15%20*%20*%20*%20*) even uses this syntax, i.e. `20/15`, which crontab.guru also complains about. Maybe the guru is wrong here. (It happily accepts `0-0/15`)

Frankly, I have no idea. I just saw the problem there. I asked ChatGPT too and it too mentioned that it's not recommended to use 0/15

**Comment by nicktrn** [comment]
Certainly a surprising result. Good practice to verify in any case, keeps everyone on their toes. Thanks again :pray:

**Pull request merged**